.github/workflows/release.yml

```
@@ -8,45 +8,45 @@
+          body: |
+            Changes in this Release
+            - Auto-generated release notes
+          draft: false
+          prerelease: false
+
+      - name: Upload Release Assets
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
```

actionlint (1.7.7)

property "create_release" is not defined in object type {}

(expression)